src/components/ArticleComments.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react";
+import { fetchCommentsById } from "../../api";
+import CommentCard from "./CommentCard";
+
+function ArticleComments({ id }) {
+  const [comments, setComments] = useState([]);
+
+  function getCommentsById() {
+    fetchCommentsById(id).then((commentsData) => {
+      const commentsInfo = commentsData.data.comments;
+      setComments(commentsInfo);
+    });
+  }
+
+  useEffect(getCommentsById, []);
```

Comment:
It might be good practice to put the id in the dependency array, a change in Id should refetch the comments with the new id :)